### DISCUSSION TITLE
[A-L] (8 Febbraio 2022) Esercizio 6

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#360] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/153036952-bc1ecbd1-9396-419e-8f07-072a5b124ca2.png)

**Comment by i0nut02** [comment]
{-q,r},{-r},{-q},{-q,q},{-q},{p,q},{p},{-p}, = unsat

**Comment by matypist** [comment]
Confermo unsat:

![image](https://user-images.githubusercontent.com/69389461/153164085-4d58ec9a-0414-415b-a289-2443c9586d93.png)
